src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/tableView.js

```diff
@@ -0,0 +1,249 @@
+            });
+            result += "</tr>";
+            return tableOf(result, level);
+        }
+
+        var toTableCell = function (data, level) {
+            if (Array.isArray(data)) {
+                return "<td>" + genTable(data, level + 1) + "</td>";
+            } else if (data instanceof Object) {
+                return "<td>" + genRowObjectTable(data, level + 1) + "</td>";
+            } else {
+                if (data === undefined || data === null) data = "";
+                var res = data.toString();
+                return '<td class="plaintext">' + (res === "" ? "N/A" : res) + '</td>';
+            }
+        }
+
+        var genTable = function (data, level, header) {
+            if (isMatrix(data)) {
+                return genMatrix(data, level, header);
+            } else if (isObjectMatrix(data)) {
+                return genObjectMatrix(data, level);
+            } else {
+                return genGenericTable(data, level);
+            }
+        }
+
+        while (this.dom.firstChild) {
+            this.dom.removeChild(this.dom.lastChild);
+        }
+
+        const style_dark = `
+        <style>
+        table {
+            font-family: sans-serif;
+            font-size: 12px;
+        }
+        
+        td {
+            color: rgba(255, 255, 255, 0.9);
+            padding: 0;
+        }
+        
+        td.plaintext,
+        th {
+            padding: 5px;
+        }
+        
+        th,
+        td {
+            border: 1px solid transparent;
+            background-clip: padding-box;
+        }
+        
+        th {
+            color: rgba(255, 255, 255, 0.7);
+            font-weight: 400;
+        }
+        
+        td,
+        th {
+            background-color: rgba(255, 255, 255, 0.03);
+        }
+        </style>
+        `;
+
+        const style_light = `
+        <style>
+        table {
+            font-family: sans-serif;
+            font-size: 12px;
+        }
+
+        td {
+            color: rgba(0, 0, 0, 0.9);
+            padding: 0;
+        }
+
+        td.plaintext,
+            th {
+            padding: 5px;
+        }
+
+        th,
+            td {
+            border: 1px solid transparent;
+            background-clip: padding-box;
+        }
+
+        th {
+            color: rgba(0, 0, 0, 0.7);
+            font-weight: 400;
+        }
+
+        td,
+            th {
+            background-color: rgba(0, 0, 0, 0.03);
+        }
+        </style>`
+
+        const width   = this.dom.getAttributeNS(null, "width");
+        const height  = this.dom.getAttributeNS(null, "height");
+        const tabElem = document.createElement("div");
+        tabElem.setAttributeNS(null,"id"     ,"vis-tbl-view");
+        tabElem.setAttributeNS(null,"viewBox","0 0 " + width + " " + height);
+        tabElem.setAttributeNS(null,"width"  ,"100%");
+        tabElem.setAttributeNS(null,"height" ,"100%");
+        const tblViewStyle = `width: ${width-10}px;
```

Comment:
move to next line and use 4 spaces indent instead

Reply:
ofc!